plugins/utils/auto-save.nix

```diff
@@ -73,72 +102,57 @@ in
             local fn = vim.fn
             local utils = require("auto-save.utils.data")
 
-            if
-              fn.getbufvar(buf, "&modifiable") == 1 and utils.not_in(fn.getbufvar(buf, "&filetype"), {}) then
-              return true -- met condition(s), can save
+            if utils.not_in(fn.getbufvar(buf, "&filetype"), {}) then
+              return true
             end
-            return false -- can't save
+            return false
           end
```

Comment:
This first value should be the plugin default. (same feedback as the test)

You could do this to define an example, though:

```nix
condition = helpers.defaultNullOpts.mkLuaFn' {
  pluginDefault = null;
  description = ''
    Function that determines whether to save the current buffer or not.
    - return true: if buffer is ok to be saved
    - return false: if it's not ok to be saved

    In this example, the second argument of `utils.not_in(..., {})`
    determines which filetypes will be ignored by auto-save plugin.
    Buffers that are `nomodifiable` are not saved by default.
  '';
  example = ''
    function(buf)
      local fn = vim.fn
      local utils = require("auto-save.utils.data")
      if
        fn.getbufvar(buf, "&modifiable") == 1 and utils.not_in(fn.getbufvar(buf, "&filetype"), {}) then
        return true -- met condition(s), can save
      if utils.not_in(fn.getbufvar(buf, "&filetype"), {}) then
        return true
      end
      return false -- can't save
      return false
    end
  '';
};
```

Reply:
Yes. This will be better. Done.